reference/SUPPORTED_QUERIES.md

```diff
@@ -0,0 +1,95 @@
+# EQL Supported Queries
+
+`eqlpy` supports querying on encrypted data through EQL's [specialized functions](https://github.com/cipherstash/encrypt-query-language#searching-data-with-eql).
+
+## Table of contents
+
+- [Query types]
+- [Required index types]
+- [Supported interfaces]
+
+## Query types
+
+`eqlpy` supports X types of queries. They are:
+
+- Equality where values are exactly matched, like `X = Y`
+- Partial matching where a string is a substring of another, similar to `X like Y`
+- Comparison where a value is greater than, less than, or equal to another, like `X > Y`
+- JSONB containment where a document is contained in another, like `X @> Y`
+- JSONB value extraction where a value is extracted from a document with a path, like `$.X.Y`
```

Comment:
Should we include `ORDER BY` queries?

Reply:
I think it may be good to add as a sub item of comparison:
```
- Comparison where a value is greater than, less than, or equal to another, like `X > Y`
  - Ordering is supported where comparison is supported, like `ORDER BY C`
```